project2/src/store/Slice/cartSlice.js

```diff
@@ -26,8 +24,29 @@ const cartReducer = createSlice({
 
       localStorage.setItem("cart", JSON.stringify(state));
     },
+
+    updateQuantity: (state, action) => {
+      const { products } = state;
+      const { id, quantity } = action.payload;
+      const productIndex = products.findIndex((item) => item.id === id);
+
+      products[productIndex].quantity = quantity;
+      localStorage.setItem("cart", JSON.stringify(state));
```

Comment:
Thông thường thì ngta sẽ lưu trên localStorage như này nè. Tìm hiểu xem sao nhé
https://stackoverflow.com/questions/67248867/react-redux-toolkit-how-can-i-can-store-my-initialstate-to-localstorage-using-c